docs/src/main/sphinx/client/jdbc.md

```diff
@@ -21,7 +21,10 @@ Versions before 350 are not supported.
 
 Download {maven_download}`jdbc` and add it to the classpath of your Java application.
 
-The driver is also available from Maven Central:
+The driver is available from Maven Central as a full JAR archive that contains
```

Comment:
s/full/shaded ?

Reply:
Lots of people wont know what "shaded" means

Reply:
fat jar then?

Reply:
Sure .. kinda weird term but it is more commonly used.

Reply:
Updated now.